projects/vdk-plugins/vdk-jupyter/.plugin-ci.yml

```diff
@@ -0,0 +1,34 @@
+# Copyright 2021-2023 VMware, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+.retry:
+  retry_options:
+    max: 1
+    when:
+      - always
+
+
+build-vdk-jupyterlab-extension:
+  stage: build
+  image: "python:3.8-alpine"
+  before_script:
+    - cd projects/vdk-plugins/vdk-jupyter/vdk-jupyterlab-extension || exit 1
+    - pip install -U pip setuptools pre-commit
+    - apk --no-cache add npm
+    - apk add rust cargo
+    - apk add libffi-dev
+    - pip install pytest-cov
```

Comment:
Why did you decide to install pytest-cov in the before_script section and then jupyter in the script section?

Reply:
@murphp15 actually, I will delete the pytest-cov now since it is not needed here we only check the installation and the build, will move the jupyter installation